sources/europe/KantonZuerich10cm-2015-TMS.json

```diff
@@ -0,0 +1,457 @@
+{
+    "url": "http://mapproxy.osm.ch:8080/tiles/KTZUERICH2015/EPSG900913/{z}/{x}/{y}.png?origin=nw",
+    "attribution": {
+        "text": "Kanton Zürich OF 2015",
+        "required": false
+    },
+    "type": "tms",
+    "name": "Kanton Zürich 2015  10cm",
+    "country_code": "CH",
+    "best": true,
```

Comment:
This source seems to overlap with [Stadt Zürich Luftbild 2011](https://github.com/osmlab/editor-layer-index/blob/b3c1e78c6fdfce260e8a208fc8fa639c15f63492/sources/europe/StadtZuerichOrthophoto2011.json#L10), which is also marked `best: true`.  Should the other one be removed?